offchain/log/src/lib.rs

```diff
@@ -10,7 +10,7 @@ pub mod built_info {
     include!(concat!(env!("OUT_DIR"), "/built.rs"));
 }
 
-#[derive(Debug, Parser)]
+#[derive(Debug, Parser, Clone)]
```

Comment:
I'd consider whether the cases you're cloning the config could instead just borrow.